deploy/bundle/catalogsource.yaml

```diff
@@ -0,0 +1,262 @@
+                  - apps
+                  resources:
+                  - replicasets
+                  - deployments
+                  verbs:
+                  - get
+                - apiGroups:
+                  - wmc.openshift.io
+                  resources:
+                  - '*'
+                  verbs:
+                  - create
+                  - delete
+                  - get
+                  - list
+                  - patch
+                  - update
+                  - watch
+            deployments:
+              - name: windows-machine-config-operator
+                spec:
+                  replicas: 1
+                  selector:
+                    matchLabels:
+                      name: windows-machine-config-operator
+                  template:
+                    metadata:
+                      labels:
+                        name: windows-machine-config-operator
+                    spec:
+                      containers:
+                        - image: PUT_IMAGE_HERE
+                          imagePullPolicy: Always
+                          name: windows-machine-config-operator
+                          env:
+                            - name: WATCH_NAMESPACE
+                              valueFrom:
+                                fieldRef:
+                                  fieldPath: 'metadata.annotations[''olm.targetNamespaces'']'
+                            - name: POD_NAME
+                              valueFrom:
+                                fieldRef:
+                                  fieldPath: metadata.name
+                            - name: OPERATOR_NAME
+                              value: windows-machine-config-operator
+                      nodeSelector:
+                        beta.kubernetes.io/os: linux
+                      serviceAccountName: windows-machine-config-operator
+        installModes:
+          - type: OwnNamespace
+            supported: true
+          - type: SingleNamespace
+            supported: true
+          - type: MultiNamespace
+            supported: false
+          - type: AllNamespaces
+            supported: false
+  customResourceDefinitions: |
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: windowsmachineconfigs.wmc.openshift.io
+      spec:
+        group: wmc.openshift.io
+        names:
+          kind: WindowsMachineConfig
+          listKind: WindowsMachineConfigList
+          plural: windowsmachineconfigs
+          singular: windowsmachineconfig
+        scope: Namespaced
+        subresources:
+          status: {}
+        validation:
+          openAPIV3Schema:
+            description: WindowsMachineConfig is the Schema for the windowsmachineconfigs API
+            properties:
+              apiVersion:
+                description: >-
+                  APIVersion defines the versioned schema of this representation of an
```

Comment:
I think you need to update this to talk about our api?

Reply:
Im pretty sure this just describes the apiVersion field